app/services/notifications/remove_each.rb

```diff
@@ -0,0 +1,27 @@
+module Notifications
+  class RemoveEach
+    def initialize(notifiable_collection, action)
+      @notifiable_collection = notifiable_collection
+      @action = action
+    end
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+    def call
+      # only used for mentions since it's an array
```

Comment:
this comment can be removed because it's not true anymore

Reply:
Agree, and I also suggest adding a comment on what `notifiable_collection` and `action` can be.

Reply:
I looked at the code a bit more, and I see that `remove_each` is only used without action, so the `action`  parameter can be removed.

Reply:
I have removed the comment, and therefore refactored to not rely only on Mention object. I have removed action and added comments about notifiable.